devicetypes/smartthings/zigbee-valve.src/zigbee-valve.groovy

```diff
@@ -140,3 +141,11 @@ def configure() {
     log.debug "Configuring Reporting and Bindings."
     refresh()
 }
+
+def installed() {
+    sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+}
+
+def ping() {
+    refresh()
```

Comment:
`refresh` sends a lot of commands. We generally want `ping` to only send one or two commands since injecting a bunch of traffic into the network unnecessarily can degrade performance. I suggest changing this to just `zigbee.onOffRefresh()`